tests/python_tests/test_vlm_pipeline.py

```diff
@@ -269,34 +269,36 @@ def test_sampling(config, cache):
     pipe.generate(prompts[0], image=image, generation_config=config)
 
 
-@pytest.mark.skip(reason="CVS-160580: tests/python_tests/test_vlm_pipeline.py::test_perf_metrics fails")
 @pytest.mark.precommit
 @pytest.mark.nightly
 def test_perf_metrics(cache):
     import numpy as np
+    import datetime
     models_path = get_ov_model("katuni4ka/tiny-random-minicpmv-2_6", cache)
 
     images = [get_image_by_link(image_links[0])]
 
+    start_time = datetime.datetime.now()
     pipe = VLMPipeline(models_path, "CPU")
     result = pipe.generate(prompts[0], images=images, generation_config=GenerationConfig(max_new_tokens=30))
+    generate_time = (datetime.datetime.now() - start_time).total_seconds() * 1000
```

Comment:
Use https://docs.python.org/3/library/time.html#time.perf_counter_ns instead. I don't think `datetime` is monotonic

Reply:
Done.